ansible_roles/roles/azure_create/tasks/main.yml

```diff
@@ -89,8 +89,11 @@
 - name: copy general tf files
   shell: "cat {{ working_dir }}/add_vars_tf >> {{ working_dir }}/tf//vars.tf"
 
-- name: Add tag info in.
-  shell: "cat {{ working_dir }}/add_main_vars.tf >> {{ working_dir }}/tf/main.tf"
+- name: Move main.tf off to the side
+  shell: "mv {{ working_dir }}/tf/main.tf {{ working_dir }}//main_back.tf"
```

Comment:
Couldn't we just rename `main.tf` to something else?  Or is it needed before these steps in the playbook?

Reply:
If it is, fix the double `/`, will start looking for those.

Reply:
Renaming main.tf would impact AWS and GCP cloud pieces also, thus the moving things around.
// has been corrected.